README.md

```diff
@@ -1,13 +1,36 @@
 [![Type Hero Logo](/logo.svg)](https://typehero.dev/)
 
+<!-- do we want badge placeholders?? -->
+
 # TypeHero
 
-Connect, collaborate, and grow with a community of TypeScript developers. Elevate your skills through interactive coding challenges, discussions, and knowledge sharing
+Connect, collaborate, and grow with a community of TypeScript developers.
+Elevate your skills through interactive coding challenges, discussions, and
+knowledge sharing
+
+## About Us
+
+TypeScript's typing system can be very powerful when used correctly. The problem
```

Comment:
nice. is that chatgpt 🤣

Reply:
no, just autism 🙂 figured we could tweak it.